components/header/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import Link from "next/link";
+import {
+  Content,
+  DateNow,
+  DateToDay,
+  HeaderStyles,
+  IconSearchStyles,
+  MobileDate,
+  SearchBlock,
+  Text,
+  Title
+} from "./header.styled";
+import {HeaderProps} from "../../interfaces/header";
+import {Container} from "../../styles/global.styled";
+import SearchIcon from "../../assets/icons/search.svg"
+
+const Header: React.FC<HeaderProps> = ({setIsPopup, isPopup}) => {
+
+  const date = new Date()
+  const monthNames = ["Jan", "Feb", "Mar", "Apr", "May", "Jun",
+    "Jul", "Aug", "Sep", "Oct", "Nov", "Dec"
+  ];
+  const day = date.getDate()
+  const month = date.toLocaleString(
```

Comment:
usually format date logic is moved to helpers/utilities, because it is used all around the project and usually have same format. So even if for this project it is used in one place - it always make sense to move it to helper. Pls try to create such helper and reuse it here